src/Card/CardSection/CardSectionContent/index.js

```diff
@@ -54,7 +54,7 @@ class CardSectionContent extends React.Component<Props, State> {
     contentHeight: 0,
   };
 
-  node: { current: any | HTMLDivElement } = React.createRef();
```

Comment:
This is wrong everywhere you use `{ current: any | Something }`, see: https://github.com/facebook/flow/blob/master/lib/react.js#L246-L247. I left it everywhere there it uncovered additional bugs. Maybe worth checking it.